src/main/java/dev/j3fftw/litexpansion/NanoBladeActiveEnchant.java

```diff
@@ -53,7 +56,14 @@ public boolean conflictsWith(Enchantment enchantment) {
     }
 
     @Override
-    public boolean canEnchantItem(ItemStack itemStack) {
-        return SlimefunUtils.isItemSimilar(itemStack, Items.NANO_BLADE, false);
+    public boolean canEnchantItem(ItemStack item) {
+        if (item.hasItemMeta()) {
+            ItemMeta itemMeta = item.getItemMeta();
+            Optional<String> id = SlimefunPlugin.getItemDataService().getItemData(itemMeta);
+            if (id.isPresent()) {
+                return (id.get().equals((Items.NANO_BLADE).getItemId()));
+            }
+        }
```

Comment:
why? isItemSimilar will do an ID check if there's an ID to check (which there always will be for LX)

Reply:
Its if someone tries shoving the active enchant into an auto enchanter with a vanilla sword

Reply:
isItemSimilar will handle that. You're doing the exact same as isItemSimilar would

Reply:
oh i remember why i did this now, its for netherite

Reply:
isItemSimilar compares the materials, I chopped that part off with some other unnecessary bits so now it just compares the IDs

Reply:
But once again, it will always compare ID if it uses IDs which it always does with LX.